packages/features/eventtypes/lib/types.ts

```diff
@@ -65,9 +65,11 @@ export type FormValues = {
     beginMessage: string;
     yourPhoneNumber: string;
     numberToCall: string;
-    guestName: string;
-    guestEmail: string;
-    guestCompany: string;
+    guestName?: string;
+    guestEmail?: string;
+    guestCompany?: string;
+    templateType: string;
+    schedulerName?: string;
```

Comment:
I'm not sure how I feel about declaring these fields in `formValues`. In the future do we want users to set their own fields? If so maybe we should look at how the `customInputs` are typed.

Reply:
> In the future do we want users to set their own fields?

Yeah I have mentioned this in PR description. In the follow up PR I would add a button to add custom fields and it would be of format like TEMPLATES_FIELDS value

```
  {
      type: "textarea",
      name: fieldNameEnum.enum.generalPrompt,
      required: true,
      defaultLabel: "general_prompt",
      placeholder: "Enter your prompt",
    },
    
 ```